paimon-hive/paimon-hive-catalog/src/main/java/org/apache/paimon/hive/HiveCatalog.java

```diff
@@ -502,6 +485,24 @@ private void updateHmsTable(Table table, Identifier identifier, TableSchema sche
         }
         table.setSd(sd);
 
+        // set table parameters
+        Map<String, String> parameters =
+                schema.options().entrySet().stream()
+                        .filter(entry -> entry.getKey().startsWith(HIVE_PREFIX))
+                        .collect(
+                                Collectors.toMap(
+                                        entry -> entry.getKey().substring(HIVE_PREFIX_LENGTH),
+                                        Map.Entry::getValue,
+                                        (v1, v2) -> v2));
+        table.setParameters(parameters);
```

Comment:
How about previous parameters in the table?

Reply:
Already modified, keep the previous parameters when modifying the table.